docker/volumeplugin/exec.go

```diff
@@ -10,7 +10,7 @@ import (
 
 const (
 	statusSuccess      = "Success"
-	statusFailure      = "Failed"
```

Comment:
@jimengliu Can you update the `common.sh` as I see that is what is really wrong.